apps/src/dance/lab2/views/DanceView.tsx

```diff
@@ -81,6 +91,20 @@ const DanceView: React.FunctionComponent<
     [dispatch]
   );
 
+  const saveProject = (selectedSong: string, forceSave = false) => {
+    if (!workspace.current) {
+      return;
+    }
+    const blocksJson = Blockly.serialization.workspaces.save(workspace.current);
+    const sourcesToSave = {
+      selectedSong,
+      source: blocksJson,
```

Comment:
Making the explicit call here to save Blockly sources as direct JSON instead of stringified JSON (like we do in legacy labs). This is something I've been wanting to do for Music Lab as well, and reduces the need for extra parsing/stringification on either end.

Reply:
Hmm good question, maybe in the future? in which case I guess we'd need to update this format, but that would also happen if this was saved a string I'd assume